apidoc/v2/openapi.yaml

```diff
@@ -1364,10 +1364,26 @@ paths:
                         'exclusiveeditmode',
                         'lock',
                         'pin',
+                        'sign',
                         'timestamp',
                         'updatemetadatafield',
                         'update',
                       ]
+                      default: 'sign'
+                    passphrase:
+                      type: string
+                      default: 'your-passphrase'
+                      description: Required when action is 'sign'
```

Comment:
Description field should describe the field: use: `The passphrase corresponding to your signature key. This parameter is required when Action is "sign"`